test/object_store/run

```diff
@@ -62,6 +63,40 @@ def check_with_cql(ip, ssl):
     return checker
 
 
+def run_with_dir(run_cmd_gen, run_dir):
+    print(f'Start scylla (dir={run_dir}')
+    mask = signal.pthread_sigmask(signal.SIG_BLOCK, {})
+    signal.pthread_sigmask(signal.SIG_BLOCK, {signal.SIGINT, signal.SIGQUIT, signal.SIGTERM})
+    pid = os.fork()
+    if pid == 0:
+        # child
```

Comment:
The original cql-pytest's code collected all running scylla pids and tempdirs in a global variable here and cleaned it all on abnormal stop. How does it work in this case?

Reply:
in this case, the lifecycle of scylla launched by the test itself is always managed by `managed_cluster()`. so `kill_with_dir()` is always called. and the temporary directory is managed by https://docs.pytest.org/en/latest/reference/reference.html#tmp-path. so it should help us clean things up.